docs/swagger.json

```diff
@@ -928,68 +911,83 @@
-          "originator_id" : {
-            "type" : "string",
-            "description" : "Originator/bank-number for end user's bankID, valid BankID ODS number",
-            "example" : "9980"
+          "error" : {
+            "type" : "string"
           },
-          "nnin" : {
+          "status" : {
             "type" : "string",
-            "description" : "End user's Norwegian national identification number, string of 11 digits",
-            "example" : "11111111016"
-          },
-          "activation_id" : {
+            "enum" : [ "unknown_subject", "internal_error" ]
+          }
+        },
+        "description" : "The result of asking an ra to notify a user of activation, used in case of some error"
+      },
+      "ActivationMetadataDTO" : {
+        "type" : "object",
+        "properties" : {
+          "flow" : {
             "type" : "string",
-            "description" : "The id of this activation attempt, used for for logging, will be the same for both codes",
-            "format" : "uuid"
+            "description" : "The flow used for activation, meanings:<ul><li> - activation_code: A single activation code provided over a secure channel. </li><li> - selfservice: Activation codes provided over two insecure channels. </li><li> - bankid_auth: Activation through a BankID authentication. </li></ul>",
+            "example" : "selfservice",
+            "enum" : [ "activation_code", "selfservice", "bankid_auth" ]
           },
-          "msisdn_reference_id" : {
-            "type" : "string",
-            "description" : "The alias for the msisdn previously provided in a response from check-user"
+          "via_distribution_methods" : {
+            "type" : "array",
+            "description" : "The distribution methods (if any) used during activation.",
+            "items" : {
+              "$ref" : "#/components/schemas/ViaDistributionMethodDTO"
+            }
           },
           "locale" : {
             "type" : "string",
             "description" : "Locale of initiating app, either \"no\" for Norwegian and Norwegian dialects, otherwise \"en\"<h5>(the 'no' value is shown as 'false', openapi nuisance)</h5>",
             "example" : "en",
             "enum" : [ "no", "en" ]
           },
-          "verification_code" : {
-            "type" : "string",
-            "description" : "Four digit code"
-          },
-          "exp" : {
-            "type" : "integer",
-            "description" : "Time when code expire, ms since epoch, UTC",
-            "format" : "int64"
+          "human_readable" : {
+            "type" : "object",
+            "additionalProperties" : {
+              "type" : "string",
+              "description" : "Map of localized strings explaining how the device was activated",
+              "example" : "{\"no\":\"med aktiveringskoder gitt p� email til que***@xyz.no og p� sms til XXX XX X42\",\"en\":\"using activation codes by email to que***@xyz.no and sms to XXX XX X42\"}"
```

Comment:
hm, there are some unicode problems it seems. It was fine in the original file but garbled in this PR